contextualized/analysis/pvals.py

```diff
@@ -15,6 +15,21 @@
 from contextualized.easy.wrappers import SKLearnWrapper
 
 
+def get_pval_range(num_bootstraps: int) -> list:
```

Comment:
We should have some reference to it being potential range of pvals, not realized range.